security-analysis/security-analysis-api/src/main/java/com/powsybl/security/condition/ViolationCondition.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.condition;
+
+import static com.powsybl.security.condition.ConditionType.VIOLATION_CONDITION;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ * <p>
+ * simulate the associated action if there is a violation on any of those network elements
+ */
+public class ViolationCondition extends AbstractCondition {
+    @Override
+    ConditionType getType() {
+        return VIOLATION_CONDITION;
+    }
+}
```

Comment:
Here we would need a list of IDs.

The condition is "there is a violation on one of the network elements identified by those IDs".

Note that in internal project, it was identified as an ambiguity: do we mean "at least one", or "all" elements ?
Maybe we should be more explicit in the condition naming, like `AtLeastOneViolation`